README.md

```diff
@@ -66,3 +66,16 @@ rostest moveit_simple motoman_mh5_utest.launch -r
 ```
 
 **NOTE:** Refresh the RobotModel by un-checking and re-checking the checkbox.
+
+
+## Quick Demo
+
+1. Launch the RViz visualizer
+```
+roslaunch moveit_simple test_display.launch
+```
+
+1. Launch the demo
+```
```

Comment:
I think `test_display.launch` should be included inside `kuka_kr210_demo.launch` so the robot model panel doesn't need to be refreshed. Also is there a reason not to change the rviz config to use `base_link` as fixed frame by default?

Reply:
The motoman has a world link